kupiekota/posts/templates/posts/_post.html

```diff
@@ -0,0 +1,24 @@
+<table id="oferta">
+    <thead>
+      <th>
+        <h2 class="card-title" id="tytolpost">{{post.title}}</h2>
+      </th>
+    </thead>
+    <tr>
+      <td>
+        {% if post.post_image %}
+        <img src="{{post.post_image.url}}" width="260px" class="img-thumbnail" alt="Nie działa :(">
```

Comment:
To jest na razie spoko, ale na produkcyjnych stronach używamy narzędzia do automatycznego resizowania obrazków - bo jak wyświetlasz bezpośrednio to co ktoś wrzucił to czasem może się ładować obrazek na 500MB, bo ktoś wrzucił coś ogromnego.